Common/TableProducer/trackselection.cxx

```diff
@@ -56,8 +56,7 @@ struct TrackSelectionTask {
   void process(soa::Join<aod::FullTracks, aod::TracksExtended> const& tracks)
   {
     for (auto& track : tracks) {
-      filterTable((uint8_t)globalTracks.IsSelected(track),
-                  (uint8_t)globalTracksSDD.IsSelected(track),
+      filterTable((uint8_t)globalTracksSDD.IsSelected(track),
                   globalTracks.IsSelectedFlag(track));
```

Comment:
As an alternative to the dynamic columns which cannot be used in `Filters`, you could probably do it like this:
```
for (auto& track : tracks) {
  flagtype flags = globalTracks.IsSelectedFlag(track);
  bool isGlobalTrack = (flags & TrackSelectionFlags::kGlobalTrack) == TrackSelectionFlags::kGlobalTrack;
  filterTable((uint8_t)isGlobalTrack,
                    (uint8_t)globalTracksSDD.IsSelected(track),
                     flag);
}
```

Reply:
Indeed this is what I did at the end with the latest commit, thanks!